config/services.yaml

```diff
@@ -162,3 +162,21 @@ services:
         arguments:
             - '@OAT\Library\Lti1p3BasicOutcome\Service\Server\Handler\BasicOutcomeServiceServerRequestHandler'
         tags: [ 'controller.service_arguments' ]
+
+    OAT\SimpleRoster\Service\AwsS3\FolderSyncService:
+        arguments:
+            - '@oneup_flysystem.user_files_local_filesystem_filesystem'
+            - '@oneup_flysystem.user_files_s3_filesystem_filesystem'
+            - '%env(FILESYSTEM_USER_FILES_ADAPTER_S3_BUCKET)%'
+
+    app.s3_folder_sync:
+        alias: OAT\SimpleRoster\Service\AwsS3\FolderSyncService
+
+    Aws\S3\S3Client:
+        arguments:
+            -
+                version: 'latest'
+                region: '%env(AWS_S3_ACCESS_REGION)%'
+                credentials:
+                    key: '%env(AWS_S3_ACCESS_KEY)%'
+                    secret: '%env(AWS_S3_ACCESS_SECRET_KEY)%'
```

Comment:
Sending AWS key/secret directly is not recommended way to auth. It should be provided by IAM roles inside ec2 instance or ENV variables.
See https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/getting-started_basic-usage.html

Reply:
fixed